terraform/variables.tf

```diff
@@ -157,3 +157,19 @@ variable "monitoring_ebs_volume" {
   default     = 100
   description = "Size of monitoring instance EBS volume in GB"
 }
+
+variable "enable_logstash" {
+  type    = bool
+  description = "Enable logstash instance on validator to send logs to elasticservice"
```

Comment:
Maybe add note to description that `enable_logstash` requires `log_to_file` var also set